client/a8c-for-agencies/sections/purchases/payment-methods/payment-method-add/credit-card-fields/credit-card-submit-button.tsx

```diff
@@ -0,0 +1,111 @@
+import { Button, FormStatus, useFormStatus } from '@automattic/composite-checkout';
+import { useElements, CardElement } from '@stripe/react-stripe-js';
+import { useDispatch, useSelect } from '@wordpress/data';
+import debugFactory from 'debug';
+import { useTranslate } from 'i18n-calypso';
+import { useMemo } from 'react';
+import { creditCardStore } from 'calypso/state/partner-portal/credit-card-form';
```

Comment:
I believe we can reuse this one for the A4A use case. There is no need to build a separate state for the a4a store.